x11-packages/hicolor-icon-theme/build.sh

```diff
@@ -2,13 +2,12 @@ TERMUX_PKG_HOMEPAGE=https://www.freedesktop.org/wiki/Software/icon-theme/
 TERMUX_PKG_DESCRIPTION="Freedesktop.org Hicolor icon theme"
 TERMUX_PKG_LICENSE="GPL-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=0.17
-TERMUX_PKG_REVISION=28
-TERMUX_PKG_AUTO_UPDATE=false
-TERMUX_PKG_SKIP_SRC_EXTRACT=true
+TERMUX_PKG_VERSION=1:0.17
+TERMUX_PKG_SRCURL=https://gitlab.freedesktop.org/xdg/default-icon-theme/-/archive/${TERMUX_PKG_VERSION:2}/default-icon-theme-master.tar.gz
```

Comment:
I don't think it is necessary to bump epoch of this package. This PR only changes its source URL and will not make this package downgrade.

Reply:
Fair enough.
Let me push that real quick.